### ISSUE TITLE
[New Discover tab] Clicking x or outside the tool tip

### ISSUE THREAD

**Issue by setch-l** [planetary-social/nos#1026]
Given that I am a user of the Nos Discover tab
When I click on the x or outside the tool tip
Then the tool tip closes.

**Comment by joshuatbrown** [comment]
@setch-l as I understand it, this depends on #1025 which isn't currently in the sprint. Can we move it in?

**Comment by setch-l** [comment]
@joshuatbrown - Yes - sorry I moved it in but couldn't remember where you'd asked about it. If I didn't you can move it in.

**Comment by joshuatbrown** [comment]
No longer needed; closing since this went along with #1025.
